docs/fast_data/concepts/data_loading.mdx

```diff
@@ -468,4 +468,183 @@ Finally, you can launch the script from your command line terminal.
 
 </TabItem>
 
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+### Advanced
```

Comment:
I would avoid adding this script, since it's very similar to the one that generates trigger messages already defined in this section.  
If you want to trigger the generation of a single view, is better to generate the trigger message directly.

Reply:
This is an advanced section dedicated to the situations where you need to regenerate the single views but the single view trigger generator has custom strategies. In this specific situation without this script you cannot regenerate the single view.